### PR TITLE
libgen-desktop: Add version 1.4.1

### PR DESCRIPTION
**Pull request by zStruCat**
As its name implies, libgen-desktop is a client for library genesis.
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/libgen-desktop.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+    "version": "1.4.1",
+    "description": "Desktop for Library Gensis",
```
Comment (rashil2000) [inline bucket/libgen-desktop.json:3]:
```suggestion
    "description": "Desktop application for Library Genesis",
```

File: bucket/libgen-desktop.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "homepage": "https://github.com/libgenapps/LibgenDesktop",
+    "license": "Unknown",
+    "notes": [
+        "Require Windows Vista SP2 or later and .NET Framework 4.5 (Included in Windows 8 or later by default) or higher to run.",
+        "See also https://wiki.mhut.org/software:libgen_desktop for details."
+    ],
```
Comment (rashil2000) [inline bucket/libgen-desktop.json:9]:
```suggestion
```

This section is not required. Scoop is only supported on Windows 7 and above (which already have this).

File: bucket/libgen-desktop.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "64bit": {
+            "url": "https://github.com/libgenapps/LibgenDesktop/releases/download/1.4.1/LibgenDesktop.Portable.64-bit.zip",
+            "hash": "3560006aa20d9be09b6b37cf818364e3b7e0862313a15eb7ce3e8840b2af7c47"
+        },
+        "32bit": {
+            "url": "https://github.com/libgenapps/LibgenDesktop/releases/download/1.4.1/LibgenDesktop.Portable.32-bit.zip",
+            "hash": "2142efa5924cc0d47dc5751cd88892ca4e7dc1beb919c11739f562e94fe05c59"
+        }
+    },
+    "extract_to": "",
```
Comment (rashil2000) [inline bucket/libgen-desktop.json:24]:
```suggestion
```

File: bucket/libgen-desktop.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            "LibgenDesktop"
+        ]
+    ],
+    "persist": [
+        "Mirrors",
+        "Downloads",
+        "libgen.config",
+        "downloads.json"
+    ],
+    "checkver": "github",
```
Comment (rashil2000) [inline bucket/libgen-desktop.json:37]:
```suggestion
```

I don't think this app is going to get any updates (the github repo is abandoned). So we can remove this field.

File: bucket/libgen-desktop.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    "homepage": "https://github.com/libgenapps/LibgenDesktop",
+    "license": "Unknown",
+    "notes": [
+        "Require Windows Vista SP2 or later and .NET Framework 4.5 (Included in Windows 8 or later by default) or higher to run.",
+        "See also https://wiki.mhut.org/software:libgen_desktop for details."
+    ],
+    "pre_install": [
+        "Out-File -FilePath \"$dir\\libgen.config\" -Encoding ascii",
+        "Out-File -FilePath \"$dir\\downloads.json\" -Encoding ascii"
+    ],
```
Comment (rashil2000) [inline bucket/libgen-desktop.json:13]: The proper way to do this is like this:
https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/xmplay.json#L8-L12

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by zStruCat** [comment]
/verify
Change has been made according to review opinion

File: bucket/libgen-desktop.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "1.4.1",
+    "description": "Desktop application for Library Genesis",
+    "homepage": "https://github.com/libgenapps/LibgenDesktop",
+    "license": "Unknown",
+    "notes": [
+        "See also https://wiki.mhut.org/software:libgen_desktop for detailed functions.",
+        "For information of latest available libgen mirrors, see https://wiki.mhut.org/_media/software:libgen_desktop_mirrors_update.ziphttps://wiki.mhut.org/_media/software:libgen_desktop_mirrors_update.zip"
+    ],
```
Comment (rashil2000) [inline bucket/libgen-desktop.json:9]:
```suggestion
    "notes": "For information of latest available libgen mirrors, see https://wiki.mhut.org/_media/software:libgen_desktop_mirrors_update.zip",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Pull request merged**